multimodal/src/autogluon/multimodal/data/preprocess_dataframe.py

```diff
@@ -379,6 +389,48 @@ def transform_text(
 
         return text_features, text_types
 
+    def transform_rois(
+        self,
+        df: pd.DataFrame,
+    ) -> Tuple[Dict[str, List[List[str]]], Dict[str, str]]:
+        """
+        Preprocess image data by collecting their paths together. If one sample has multiple images
```

Comment:
Can I ask what is the use case here, i.e., why one sample has multiple images (maybe RoI)?

Reply:
Updated the descriptions, but still keep the support for multi images/rois columns. There is no current use cases, and we have limited the number of image columns to 1 in config for now. A possible future example could be we are working on videos and need multiple frames as input per entry.